ruby-challenges.rb

```diff
@@ -19,10 +35,41 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
+def sum_of_array(array)
+    array.sum
+end
+
+# p sum_of_array(nums_array1)
+# p sum_of_array(nums_array2)
 
 
 # --------------------3a) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a bike_info method that returns a sentence with all the data from the bike object.
 
+class Bike
+   
+    def initialize(model, wheels = 2, current_speed = 0)
+        @model = model
+        @wheels = wheels
+        @current_speed = current_speed
+    end
+    def bike_info
+        p "This #{@model} got #{@wheels} and it's current speed is #{@current_speed}."  
+    end
+    def pedal_faster(increase_speed)
+        @current_speed = @current_speed + increase_speed
```

Comment:
Please reconsider the structure of this equation. It should keep adding another value to the result of the previous. For instance, if the current speed is 0, and we speed up by 15 mph. then 0 + 15 = 15mph.
Then if we speed up by 23 mph, then 15 + 23 = 38 mph.